submissions/yanyshpolska/Hooli-style Popup/css/style.css

```diff
@@ -0,0 +1,213 @@
+/* base styles */
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
+  -moz-box-sizing: border-box;
+  -webkit-box-sizing: border-box;
+  box-sizing: border-box;
+}
+:focus,
+:active {
+  outline: none;
+}
+html,
+body {
+  height: 100%;
+  width: 100%;
+  font-size: 100%;
+  line-height: 1;
+  font-size: 14px;
+  -ms-text-size-adjust: 100%;
+  -moz-text-size-adjust: 100%;
+  -webkit-text-size-adjust: 100%;
+}
+a {
+  text-decoration: none;
+}
+ul li {
+  list-style: none;
+}
+/*---------------------------------------------------*/
+.main {
+  height: 100%;
+  background-color: #fff4e4;
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+}
+/* ================Menu==================== */
+.menu__items {
+  background-color: #fff4e4;
+  margin: auto;
+  font-family: Arial, Helvetica, sans-serif;
+  border: 0.15em solid #c4b4c9;
+  display: flex;
+  justify-content: flex-end;
+  align-items: center;
+}
+.menu__item {
+  display: block;
+  height: 1.715em;
+  margin: 0.715em;
+}
+/* ===============Menu text items======================= */
+.menu__item_text {
+  /* display: block; */
+  border: 0.15em solid transparent;
+  border-radius: 0.2em;
+  color: #555;
+  padding: 0.2em 0.15em;
+}
+.menu__item_text:focus {
+  border-color: #876991;
+}
+.menu__item_text:hover {
+  border-color: #c4b4c9;
+}
+/* ===============Menu picture items==================== */
+.menu__item_picture {
+  /* display: block; */
+  width: 1.715em;
+  /* height: 1.715em; */
+  border: 0.15em solid transparent;
+  border-radius: 0.2em;
+  padding: 0.15em;
+}
+.menu__item_picture_img {
+  width: 100%;
+  height: 100%;
+  object-fit: cover;
+}
+.menu__item_picture:focus {
+  border-color: #876991;
+}
+.menu__item_picture:hover {
+  border-color: #c4b4c9;
+}
+/* ===============Menu checkbox item===================*/
+.menu__item_checkbox {
+  position: relative;
+  min-width: 1.715em;
+}
+.menu__item_checkbox_label {
+  position: absolute;
+  height: 1.715em;
```

Comment:
Try to use also rem :) 
https://essential-addons.com/elementor/css-design-guide-difference/
https://www.quora.com/What-is-the-difference-between-em-rem-and-px-in-CSS

Reply:
Please read the articles (these or any others) and add at least a couple of rems, to the corresponding classes. `Rem` is more used nowadays than `em`